episodes/05-profiling-your-MARC-data.md

```diff
@@ -170,6 +170,19 @@ The results list will show one record missing a 245 field. Click on Jump to Reco
 
 ::::::::::::::::::::::::::::::::::::::::::::::::::
 
+## Validation and Log Reports
```

Comment:
I'm wondering whether we should break these out into the steps to run them like the follow along exercises above. If not, we may be able to combine the descriptions for ISBN and ISSN reports. I'd also consider moving the Manage Log file overview to the end of the lesson.